includes/start-level.script

```diff
@@ -0,0 +1,13 @@
+bundle:start-level org.openhab.core.model.thing 83
```

Comment:
This shoudn't be specific to openHABian, nor should we be changing the behaviour of openHAB compared to other instalation types. 

If this improves the startup of openHAB then it should be suggested as a change in openHAB-distro or openHAB-core instead.

Reply:
Yep. That's what I already did. But Kai and Wouter opted against it so I've removed it here again.
Kai left me with some hope there's going to be a solution mid-term.
https://github.com/openhab/openhab-core/issues/1467#issuecomment-626358060